python/lsst/cell_coadds/_fits.py

```diff
@@ -313,8 +313,10 @@ def readAsMultipleCellCoadd(self) -> MultipleCellCoadd:
                 ), "Aperture correction map is not available for all cells."
                 aperture_correction_grid = self._readApCorr(aperture_correction_hdu, grid_shape)
             except KeyError:
-                logger.info("Unable to read aperture correction map from the file.")
-                aperture_correction_hdu = EMPTY_AP_CORR_MAP
```

Comment:
This is not the guard that I wanted to put, but `aperture_correction_grid`, which I do in this PR.